composer.json

```diff
@@ -32,7 +32,10 @@
         "symfony/symfony": "2.6.*",
         "willdurand/hateoas-bundle": "0.3.*",
         "symfony/swiftmailer-bundle": "~2.3",
-        "pagerfanta/pagerfanta": "~1.0"
+        "pagerfanta/pagerfanta": "~1.0",
+        "sulu-io/document-manager": "dev-master",
```

Comment:
I will merge into this branch. The other branch is `structure_manager_2_search`.